src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/Example8.java

```diff
@@ -0,0 +1,33 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="IllegalType" />
+    <module name="IllegalType">
+      <property name="illegalClassNames" value="var"/>
+      <property name="tokens" value="VARIABLE_DEF"/>
+      <property name="id" value="IllegalTypeVarAsField"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.illegaltype;
+
+// xdoc section -- start
+public class Example8 {
+  public void var() {
+    // violation below 'Usage of type 'var' is not allowed'
+    var message = "Hello, World!";
```

Comment:
this Check uses "the same"  code in all examples, please just add one `var ....` to all example classes.
and use in Example8 same code as in other examples, only comments should in diff between examples.